# Introduction

The principal electrical resistance can be applied through the use of resistors,
some of the most versatile components in circuitry, with applications ranging
from reduction of wattage, distribution of voltages to directing electrical
current towards desired components. Within this lab we intend to explore
electrical resistance as a concept and the equations that predict its behavior.
We will investigate the relationship between the cross-sectional area, length
and resistance, alongside determining the impact that voltage can have on
non-dedicated resistors.



In [5]:
from math import *

import matplotlib.pyplot as plt
import numpy as np
from scipy.optimize import curve_fit


# Problem: One Minute Sketch, One Minute Math

**Materials:** Resistor, calipers.

**Method:** The dimensions of the resistor were measured. The resistor was drawn
according to the measurements. Use the dimensions to calculate the resistance,
then compare to other resistors.

![Resistor](img_1.png)

Calculations for resistivity:

$$ A = \pi r^2 $$

$$ \rho = R\frac{A}{l} \implies \rho = R\frac{\pi r^2}{l} $$



In [6]:
radius = (3.9 / 2)*1e-3 # m, eye-balled "average" radius
length = 10.7e-3 # m
resistance = 1 # ohm

rho = resistance * pi * (radius ** 2) / length
print(f"{round(rho * 1e3, 3)} ohm-mm")

1.116 ohm-mm


## Data and Analysis

The number gathered, 1.116 ohm-mm, seems reasonable. The resistor is millimeters
in size, has a resistance of 1 ohm, and, similarly, a predicted resistivity of
~1 ohm-mm.

# Problem: What's the Skinny on Resistance

**Materials:** Paper resistors, desktop power supply, voltmeter.

**Method:** Consider each resistor in terms of B, where the length and height are
both equal to 1. Apply the formula for resistance for each resistor, assuming B
has a resistance of 1 ohm. Consider the predictions in terms of each other. Form
a prediction for the current if subjected to a voltage of 9V.

Set up the power supply and power it to 9V, Using the voltmeter, take
measurements at each point predicted. Compare these measurements to the
predicted values.

The ratios gathered from the size of the paper resistors:

|   | $A$ | $l$ |
|---|-----|-----|
| D | 2   | 2   |
| C | 2   | 1   |
| B | 1   | 1   |
| A | 1   | 2   |

The lab document states that the resistance of $R_C = 1 \Omega$. Finding the ratios:

$$ R = \frac{l}{A}\rho \implies R \propto \frac{l}{A} $$

$$ R_D = 1, \; R_C = \frac{1}{2}, \; R_B = 1, \; R_A = 2 $$

And multiplying the ratios to match the known value of $R_C$, our prediction is:

$$ R_D = 2\Omega, \; R_C = 1\Omega, \; R_B = 2\Omega, \; R_A = 4\Omega $$

$$ \boxed{R_A > R_B = R_D > R_C} $$

The equivalent resistance is the sum of the resistors in series. Next, a voltage
divider to find the voltage across each resistor:

$$ V_R = V \frac{R}{R_{eq}} \qquad R_{eq} = (2 + 1 + 2 + 4) \Omega = 9 \Omega $$

Assuming $V = 9 \mathrm{V}$:

$$ \boxed{V_D = 2 \mathrm{V}, \; V_C = 1 \mathrm{V}, \; V_B = 2 \mathrm{V}, \; V_A = 4
\mathrm{V}} $$

The current can be found with Ohm's law and the equivelent resistance:

$$ I = \frac{V}{R_{eq}} \implies \boxed{I = 1 \mathrm{A}} $$

## Data and Analysis

The voltages each of the nodes compared to the estimated voltage drops:

|          | $V_i$ | $V_{AB}$ | $V_{BC}$ | $V_{CD}$ | $V_{f}$ |
|----------|-------|----------|----------|----------|---------|
| Est.     | 9     | 5        | 3        | 2        | 0       |
| Measured | 8.9   | 3.3      | 2        | 1.12     | 0       |

Our measured current was: unknown --- technical difficulties. (Some of the other
groups got between 60-80 μA.)

The voltages at each node are in the same order from greatest to zero, but the
values are each off by 1-2V. This could be due to the paper not being completely
uniform, meaning the resistivity varied throughout and thus the resistance.



# Problem: Playdoh Power

**Materials:** Playdoh, conductive plates, desktop power supply, voltmeter, ammeter

**Method:** Clean the conductive brackets, if needed. Mold playdoh into a
cylindrical, makeshift resistor, then attach the conductive brackets to each
end. Take note of the dimensions of the resistor. Attach the probes for the
ammeter into the resistor, measure the distance between each probe. Set the
power supply to 9V and take measurements for voltage and current.

$$ \rho = R\frac{A}{l} \qquad R = \frac{V}{I} $$

$$ \boxed{\rho = \frac{V\pi r^2}{Il}} $$



In [7]:
length = 0.115 # m
radius = 0.055 / 2 # m
voltage = 9 # V
current = 24.2e-3 # A

rho = voltage * pi * (radius ** 2) / (current * length)

print(f"{round(rho, 3)} ohm-m")


7.683 ohm-m


## Data and Analysis

The playdoh is around 7000 times as resistive as the 1 ohm resistor from problem
(1). This makes sense as the playdoh is an insulator, unlike the more conductive
resistor designed specifically for electronics.

# Conclusion

Understanding the concept of electrical resistance is essential for electrical
engineering. Resistors have countless applications and are essential components
of circuits. Through this lab we investigated the relationship between
cross-sectional area, length and resistance across a variety of resistors
ranging from dedicated resistors where we applied these formulas to calculate
the resistivity of our resistor to be 1.116 ohm-mm, the improvised paper
resistors, where we measured the voltage drops to be similar to our predictions,
(e.g. 3.3 V, etc.) and finally our playdoh resistor, where our measured
resistance, 7.683 ohm-m, aligned with our expectations. 



